WalletWasabi.Fluent/ViewModels/TransactionBroadcasting/BroadcastTransactionViewModel.cs

```diff
@@ -85,7 +87,14 @@ private void ProcessTransaction(SmartTransaction transaction, Network network)
 		var networkFee = totalInputValue is null || totalOutputValue is null
 			? null
 			: totalInputValue - totalOutputValue;
-		FeeString = networkFee is null ? "Unknown" : $"{networkFee.ToFormattedString()} BTC";
+
+		var feeDisplayFormat = (FeeDisplayFormat?)Services.UiConfig.FeeDisplayFormat ?? FeeDisplayFormat.BTC;
```

Comment:
could the cast fail?

Reply:
@danwalmsley if the stored value is not a valid value in the enum it will return null. Pretty sure casting to a nullable enum does not throw.

I was wondering if the stored model can't hold the enum directly though, instead of `int`.

Reply:
Ok if that's the case then it's good 👍